libs/dex/src/testFixtures/java/io/crate/testing/TestingRowConsumer.java

```diff
@@ -60,8 +60,12 @@ public void accept(BatchIterator<Row> it, Throwable failure) {
     }
 
     public List<Object[]> getResult() throws Exception {
+        return getResultWithTimeout(10);
+    }
+
+    public List<Object[]> getResultWithTimeout(int timeout) throws Exception {
```

Comment:
The additional argument already documents the difference to `getResult()`
```suggestion
    public List<Object[]> getResult(int timeout) throws Exception {
```